xep-0045.xml

```diff
@@ -46,6 +46,14 @@
   <registry/>
   &stpeter;
   <revision>
+    <version>1.28</version>
+    <date>2017-05-31</date>
+    <initials>gl</initials>
+    <remark>
+      <p>Introduce &lt;x/> tag in MUC-PMs to support better Carbon delivery.</p>
```

Comment:
Nit: `&gt;` for consistency with the other side of the tag and other places tags are used in the XML